ios/FluentUI/Core/Colors2.swift

```diff
@@ -93,7 +93,7 @@ private func brandColorOverrides(provider: ColorProviding2, for theme: FluentThe
 
 // MARK: Colors
 
-public enum BrandColorsForOverriding: CaseIterable {
+private enum BrandColorsForOverriding: CaseIterable {
```

Comment:
why is it private now?

Reply:
It is not needed to be public - it should have been private from the initial PR, but it didn't make it over before the PR was merged.